sort.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+function bubbleSort($sort_array)
+{
+    $array_indexes = count($sort_array);
+    $lastIndex =  $array_indexes - 1;
+    if (strlen($sort_array[$lastIndex]) < 2) {
+        $sort_array[$lastIndex] = $sort_array[$lastIndex] . "\n";
+    }
+    for ($i = 0; $i < $array_indexes; $i++) {
+        for ($j = 0; $j < $array_indexes; $j++) {
+            if ($sort_array[$i] < $sort_array[$j]) {
+                $tmp = $sort_array[$i];
+                $sort_array[$i] = $sort_array[$j];
+                $sort_array[$j] = $tmp;
+            }
+        }
+    }
+
+    return $sort_array;
+}
+echo implode("", bubbleSort(file('php://stdin')));
```

Comment:
Unfortunately now you have one line too less. The "echo implode..." shall be the last line, but there shall not be the "No newline at and of file" icon. I.e. every line shall be \\n-terminated.